ldes-fragmentisers/ldes-fragmentisers-geospatial/src/test/resources/application-fragment-generator-test.yml

```diff
@@ -0,0 +1,9 @@
+ldes:
+  hostname: "http://localhost:8080"
+  collectionname: "mobility-hindrances"
+  shape: "https://private-api.gipod.test-vlaanderen.be/api/v1/ldes/mobility-hindrances/shape"
+  membertype: "https://data.vlaanderen.be/ns/mobiliteit#Mobiliteitshinder"
+geospatial:
+  zoomLevel: 15
+  bucketiserproperty: "http://www.opengis.net/ont/geosparql#asWKT"
+  projection: "lambert72"
```

Comment:
once again.. but then it boils down to understanding of the story rather than the code, I do not think it should be our responsibility to convert this to a universal standard.